extras/rootless/containerd-rootless-setuptool.sh

```diff
@@ -273,6 +273,50 @@ cmd_entrypoint_install_buildkit() {
 	EOT
 }
 
+# CLI subcommand: "install-buildkit-containerd"
+cmd_entrypoint_install_buildkit_containerd() {
+	init
+	if ! command -v "buildkitd" >/dev/null 2>&1; then
+		ERROR "buildkitd (https://github.com/moby/buildkit) needs to be present under \$PATH"
+		exit 1
+	fi
+	if [ ! -f "${XDG_CONFIG_HOME}/buildkit/buildkitd.toml" ]; then
+		mkdir -p "${XDG_CONFIG_HOME}/buildkit"
+		cat <<-EOF > "${XDG_CONFIG_HOME}/buildkit/buildkitd.toml"
+			[worker.oci]
+			enabled = false
+
+			[worker.containerd]
+			enabled = true
+			rootless = true
+		EOF
+	fi
+	if ! systemctl --user --no-pager status "${SYSTEMD_CONTAINERD_UNIT}" >/dev/null 2>&1; then
+		ERROR "Install containerd first (\`$ARG0 install\`)"
+		exit 1
+	fi
+	NAMESPACE="${CONTAINERD_NAMESPACE:-default}"
+	ADDRESS="${XDG_RUNTIME_DIR}/buildkit-${NAMESPACE}/buildkitd.sock"
+	ROOT="${XDG_DATA_HOME}/buildkit-${NAMESPACE}"
+	cat <<-EOT | install_systemd_unit "${NAMESPACE}-${SYSTEMD_BUILDKIT_UNIT}"
+		[Unit]
+		Description=BuildKit (Rootless)
+		PartOf=${SYSTEMD_CONTAINERD_UNIT}
+
+		[Service]
+		Environment=PATH=$BIN:/sbin:/usr/sbin:$PATH
+		ExecStart="$REALPATH0" nsenter -- buildkitd --addr="unix://${ADDRESS}" --root="${ROOT}" --containerd-worker-namespace="${NAMESPACE}" --containerd-worker-snapshotter="${CONTAINERD_SNAPSHOTTER:-overlayfs}"
```

Comment:
Just leave --containerd-worker-snapshotter unset when $CONTAINERD_SNAPSHOTTER is not set

Reply:
Same applies to other flags too